src/main/java/io/github/arrayv/sorts/misc/StalinSort.java

```diff
@@ -1,21 +1,15 @@
 package io.github.arrayv.sorts.misc;
 
 import io.github.arrayv.main.ArrayVisualizer;
+import io.github.arrayv.sortdata.SortMeta;
 import io.github.arrayv.sorts.templates.Sort;
 
+@SortMeta(
+    name = "Stalin"
+)
 public final class StalinSort extends Sort {
     public StalinSort(ArrayVisualizer arrayVisualizer) {
         super(arrayVisualizer);
-
-        this.setSortListName("Stalin");
-        this.setRunAllSortsName("Stalin Sort");
-        this.setRunSortName("Stalinsort");
-        this.setCategory("Impractical Sorts");
```

Comment:
perhaps add a category anyway to indicate it's not a real sort?

Reply:
Like `category = "Impractical Sorts"`?

Reply:
no, since those sorts still actually sort

Reply:
Sorts that don't sort are still impractical.

Reply:
"impractical" does not mean "does not work"

Reply:
It doesn't strictly mean that, but I'd say that if it doesn't work, it's impractical.
From the dictionary:
> 1. (of an object or course of action) not adapted for use or action; not sensible or realistic.
> 2. impossible to do; impracticable.

Reply:
"impossible" is not the same as "fake" - it means it would produce the desired outcome but is impossible to execute

Reply:
Stalin Sort was already in Impractical Sorts. This is, again, out of this PR's scope.

Reply:
Yeah, https://github.com/Gaming32/ArrayV/pull/86#discussion_r1060222048